src/doc/using_visit/SavingPrinting/Saving_movies.rst

```diff
@@ -190,11 +139,251 @@ click the **Movie progress dialog's** **Cancel** button.
    Movie progress dialog
 
 
-The second movie generation method will cause VisIt to save out a session file
-containing every detail about your visualization so it can be recreated by a
-new instance of VisIt. This method works well if you want to create a movie
-without sacrificing your current VisIt session but you cannot watch the movie
-as it is generated and you may have to wait for the second instance's compute
-engine to be scheduled to run. The last movie generation option simply makes
-VisIt display the command that you would have to type at a command prompt in
-order to make VisIt generate a movie of your current visualizations.
+The second movie generation method will cause VisIt_ to save out a session file containing every detail about your visualization so it can be recreated by a new instance of VisIt_.
+This method works well if you want to create a movie without sacrificing your current VisIt_ session but you cannot watch the movie as it is generated and you may have to wait for the second instance's compute engine to be scheduled to run.
+The last movie generation option simply makes VisIt_ display the command that you would have to type at a command prompt in order to make VisIt_ generate a movie of your current visualizations.
+
+
+Templated movies
+~~~~~~~~~~~~~~~~
+
+Movie templates enable you to create a set of viewports and map the contents of various visualization windows to those viewports, also incorporating simple movie transitions.
+
+To create a movie template, you can open the Save movie wizard as you normally would.
+Instead of choosing New simple movie, choose New template movie.
+VisIt will load custom user interfaces pages for the chosen movie template.
```

Comment:
Custom user interfaces pages? Or custom user interface pages? Or a custom user interfaces page?
I can't tell if it's supposed to be the way it currently is.